core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -2675,6 +2675,15 @@ secret_configs:
   - cluster: app.ci
     name: github-deploymentconfig-trigger
     namespace: ci
+- from:
+    app.ci.config:
+      attachment: sa.endurance.app.ci.config
+      bw_item: endurance_cluster
+  to:
+  - cluster_groups:
+    - build_farm
+    name: endurance-sa
+    namespace: ci
```

Comment:
You will want this to be `test-credentials`

Reply:
updated